django/apps/blue_management/blue_mgnt/templates/partials/policy_form.html

```diff
@@ -4,6 +4,14 @@
         {{ field }}
     {% endfor %}
     <table class='widget-table widget-table-details' id='policy-table'>
+	<tr>
+		<td colspan="3">
+			<i>
+				<b>Managed</b>: any user set policy will be superseded by the policy set here<br/>
+				<b>User controlled</b>: the user defines this particular item
+			</i>
+		</td>
+	</tr>
```

Comment:
They are simply tabs, not spaces. How many spaces it uses depends on the rendering.